### PR TITLE
Fix bug with chaining rescues of FastJsonapi::MultiToJson::Result

### PR DESCRIPTION
**Pull request by GarrisonD**
Fix for #129.

### CODE REVIEW

File: lib/fast_jsonapi/multi_to_json.rb [hunk 1]
```diff
@@ -33,7 +38,8 @@ def value!
 
       def rescue
         return self if ok?
-        Result.new { yield(@error) }
+
+        Result.new(*@rescued_exceptions) { yield(@error) }
```
Comment (GarrisonD) [inline lib/fast_jsonapi/multi_to_json.rb]: Actually the bug was here - after the first `rescue` only `StandardError` was catched.

**Comment by GarrisonD** [comment]
@shishirmk I hope you notice this PR 🙂 

And I think it would be nice to tell in `README.md` that users should install `oj` gem or any other JSON serializer as well to enjoy the power of `fast_jsonapi`.

**Comment by shishirmk** [comment]
@GarrisonD it does make sense to add that to the README.

File: lib/fast_jsonapi/multi_to_json.rb (hunk 1 above)
Comment (bf4) [reply lib/fast_jsonapi/multi_to_json.rb]: oops :)

```ruby
LoadError.superclass
=> ScriptError
>> LoadError.superclass.superclass
=> Exception

>> StandardError.superclass
=> Exception
```

Comment (GarrisonD) [reply lib/fast_jsonapi/multi_to_json.rb]: @bf4 oops what? 🙂 Looks like you don't know how `begin-rescue` construction works.

`rescue` catches instances of `StandardError` and its __inheritors__ but NOT __predecessors__.

So, I can't understand why you are talking about `.superclass` 🙁

Comment (bf4) [reply lib/fast_jsonapi/multi_to_json.rb]: @GarrisonD no, I know how it works. That's why I wrote oops :)

As you pointed out, rescuing StandardError doesn't rescue a LoadError. (That's the relevance of superclass. If LoadError.ancestors.include?(StandardError) there would have been no oops, right?)

Perhaps what wasn't clear was that I wrote the code with the bug and was commenting on *my* oops

Comment (GarrisonD) [reply lib/fast_jsonapi/multi_to_json.rb]: @bf4 Now everything is clear! Sorry.

**Pull request merged**